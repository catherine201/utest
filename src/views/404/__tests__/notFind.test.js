import React from 'react';
// import renderer from 'react-test-renderer';  // 这个不支持antdesign
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import notFind from '..';

Enzyme.configure({ adapter: new Adapter() });
describe('(Component) notFind -- snapshot', () => {
  it('capture snapshot of notFind', () => {
    const footer = shallow(<notFind />);
    expect(footer).toMatchSnapshot();
  });
});
