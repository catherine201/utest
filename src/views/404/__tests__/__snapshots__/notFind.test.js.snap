// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) AppSider -- snapshot capture snapshot of AppSider 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotMatch />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          404
        </h1>,
        <p>
          Page Not Found
        </p>,
        <div>
          <Button
            block={false}
            ghost={false}
            href="/"
            loading={false}
            prefixCls="ant-btn"
            type="primary"
          >
            主页
          </Button>
           
          <Button
            block={false}
            ghost={false}
            loading={false}
            onClick={[Function]}
            prefixCls="ant-btn"
          >
            返回
          </Button>
        </div>,
        <p>
          Make by Euen
        </p>,
      ],
      "className": "undefined fadeInUp",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "404",
          "className": undefined,
        },
        "ref": null,
        "rendered": "404",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Page Not Found",
          "className": undefined,
        },
        "ref": null,
        "rendered": "Page Not Found",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <Button
              block={false}
              ghost={false}
              href="/"
              loading={false}
              prefixCls="ant-btn"
              type="primary"
            >
              主页
            </Button>,
            " ",
            <Button
              block={false}
              ghost={false}
              loading={false}
              onClick={[Function]}
              prefixCls="ant-btn"
            >
              返回
            </Button>,
          ],
          "className": undefined,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "block": false,
              "children": "主页",
              "ghost": false,
              "href": "/",
              "loading": false,
              "prefixCls": "ant-btn",
              "type": "primary",
            },
            "ref": null,
            "rendered": "主页",
            "type": [Function],
          },
          " ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "block": false,
              "children": "返回",
              "ghost": false,
              "loading": false,
              "onClick": [Function],
              "prefixCls": "ant-btn",
            },
            "ref": null,
            "rendered": "返回",
            "type": [Function],
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Make by Euen",
          "className": undefined,
        },
        "ref": null,
        "rendered": "Make by Euen",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            404
          </h1>,
          <p>
            Page Not Found
          </p>,
          <div>
            <Button
              block={false}
              ghost={false}
              href="/"
              loading={false}
              prefixCls="ant-btn"
              type="primary"
            >
              主页
            </Button>
             
            <Button
              block={false}
              ghost={false}
              loading={false}
              onClick={[Function]}
              prefixCls="ant-btn"
            >
              返回
            </Button>
          </div>,
          <p>
            Make by Euen
          </p>,
        ],
        "className": "undefined fadeInUp",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "404",
            "className": undefined,
          },
          "ref": null,
          "rendered": "404",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Page Not Found",
            "className": undefined,
          },
          "ref": null,
          "rendered": "Page Not Found",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <Button
                block={false}
                ghost={false}
                href="/"
                loading={false}
                prefixCls="ant-btn"
                type="primary"
              >
                主页
              </Button>,
              " ",
              <Button
                block={false}
                ghost={false}
                loading={false}
                onClick={[Function]}
                prefixCls="ant-btn"
              >
                返回
              </Button>,
            ],
            "className": undefined,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "block": false,
                "children": "主页",
                "ghost": false,
                "href": "/",
                "loading": false,
                "prefixCls": "ant-btn",
                "type": "primary",
              },
              "ref": null,
              "rendered": "主页",
              "type": [Function],
            },
            " ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "block": false,
                "children": "返回",
                "ghost": false,
                "loading": false,
                "onClick": [Function],
                "prefixCls": "ant-btn",
              },
              "ref": null,
              "rendered": "返回",
              "type": [Function],
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Make by Euen",
            "className": undefined,
          },
          "ref": null,
          "rendered": "Make by Euen",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;

exports[`(Component) notFind -- snapshot capture snapshot of notFind 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <notFind />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {},
    "ref": null,
    "rendered": null,
    "type": "notFind",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": "notFind",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
